src/test/kotlin/no/nav/familie/ba/mottak/task/VurderLivshendelseTaskTest.kt

```diff
@@ -280,53 +286,253 @@ class VurderLivshendelseTaskTest {
+            Task.nyTask(
+                type = VurderLivshendelseTask.TASK_STEP_TYPE,
+                payload = objectMapper.writeValueAsString(
+                    VurderLivshendelseTaskDTO(
+                        PERSONIDENT_MOR,
+                        DØDSFALL
                     )
+                )
             )
+        )
+
+        assertThat(oppgaveDto.captured.beskrivelse).isEqualTo(VurderLivshendelseTask.BESKRIVELSE_DØDSFALL +
+                ": bruker og 2 barn ${PERSONIDENT_BARN} ${PERSONIDENT_BARN2}")
+
+        setupPdlMockForDødsfallshendelse(false, true, false)
+        vurderLivshendelseTask.doTask(
+            Task.nyTask(
+                type = VurderLivshendelseTask.TASK_STEP_TYPE,
+                payload = objectMapper.writeValueAsString(
+                    VurderLivshendelseTaskDTO(
+                        PERSONIDENT_BARN,
+                        DØDSFALL
+                    )
+                )
+            )
+        )
+
+        assertThat(oppgaveDto.captured.beskrivelse).isEqualTo(VurderLivshendelseTask.BESKRIVELSE_DØDSFALL +
+                ": barn ${PERSONIDENT_BARN}")
+
+        setupPdlMockForDødsfallshendelse(false, true, true)
+        vurderLivshendelseTask.doTask(
+            Task.nyTask(
+                type = VurderLivshendelseTask.TASK_STEP_TYPE,
+                payload = objectMapper.writeValueAsString(
+                    VurderLivshendelseTaskDTO(
+                        PERSONIDENT_BARN,
+                        DØDSFALL
+                    )
+                )
+            )
+        )
+
+        assertThat(oppgaveDto.captured.beskrivelse).isEqualTo(VurderLivshendelseTask.BESKRIVELSE_DØDSFALL +
+                ": 2 barn ${PERSONIDENT_BARN} ${PERSONIDENT_BARN2}")
+    }
+
+    @Test
+    fun `Skal sett riktig beskrivelsestekster når oppdater dødsfall oppgave`(){
+        every { mockSakClient.hentRestFagsakDeltagerListe(PERSONIDENT_MOR, emptyList()) } returns
+                listOf(RestFagsakDeltager(PERSONIDENT_MOR, FORELDER, SAKS_ID, LØPENDE))
+
+        every { mockSakClient.hentRestFagsak(SAKS_ID) } returns lagAktivUtvidet()
+        every { mockSakClient.hentMinimalRestFagsak(SAKS_ID) } returns lagAktivOrdinærMinimal()
+        val oppgavebeskrivelseSlot = slot<String>()
+        every { mockOppgaveClient.oppdaterOppgaveBeskrivelse(any(), capture(oppgavebeskrivelseSlot)) } returns OppgaveResponse(
+            oppgaveId = 1
+        )
+
+        setupPdlMockForDødsfallshendelse(true, false, false)
+        vurderLivshendelseTask.doTask(
+            Task.nyTask(
+                type = VurderLivshendelseTask.TASK_STEP_TYPE,
+                payload = objectMapper.writeValueAsString(
+                    VurderLivshendelseTaskDTO(
+                        PERSONIDENT_MOR,
+                        DØDSFALL
+                    )
+                )
+            )
+        )
+
+        every{ mockOppgaveClient.finnOppgaverPåAktørId(any(), any())} returns listOf(
```

Comment:
I think we need to make the code work with this:
```suggestion
        every{ mockOppgaveClient.finnOppgaverPåAktørId(mockAktørClient.hentAktørId(PERSONIDENT_BARN), any())} returns listOf(
```